sandbox/functions-recipes/includes/durable-activity-functions-with-retry.md

```diff
@@ -0,0 +1,38 @@
+## Calling activity functions with retry
+When call activities in your orchestration functions, you may want to apply a retry policy to handle transient errors that may occur. In such cases, the `DurableOrchestrationContext` provides the `CallActivityWithRetryAsync` method. The difference between `CallActivityAsync` and `CallActivityWithRetryAsync` is that the latter accepts a type of `RetryOptions` that specifies the retry behavior.
+
+```csharp
+[FunctionName("PlaceOrder")]
+public static async Task OrderOrchestration(
+    [OrchestrationTrigger] DurableOrchestrationContext context,
+    TraceWriter log)
+{
+    RetryOptions retryPolicy = new RetryOptions(
+        firstRetryInterval: TimeSpan.FromSeconds(3),
+        maxNumberOfAttempts: 3);
+
+    retryPolicy.Handle = (ex) =>
+    {
+        return (ex is InvalidOperationException) ? false : true;
```

Comment:
Is this working for you? I admittedly haven't tested this myself. Someone reported a bug indicating that the exception type exposed here is `System.Exception`, rather than what the activity function actually throws.

Reply:
I just updated the NuGet package to 1.3.1 RC. The inner exception is System.Exception. I'll update this.